plugins/destination/postgresql/client/migrate.go

```diff
@@ -139,6 +139,8 @@ func (c *Client) autoMigrateTable(ctx context.Context, table *schema.Table, chan
 			if err != nil {
 				return err
 			}
+		default:
```

Comment:
These changes were required by golangci-lint `enforce-switch-style`

Reply:
Can we disable this rule? See https://github.com/cloudquery/plugin-sdk/blob/b8cf3902daef8d1bfbde5d7c7908c70f8085c0a6/.golangci.yml#L74

Reply:
Rule disabled in 9ab172b802 but I think we should still keep the default clauses.